ansible/files/pgbouncer_config/pgbouncer.ini.j2

```diff
@@ -361,3 +361,4 @@ default_pool_size = 15
 
 %include /etc/pgbouncer-custom/generated-optimizations.ini
 %include /etc/pgbouncer-custom/custom-overrides.ini
+%include /etc/pgbouncer-custom/ssl-config.ini
```

Comment:
would this not break the docker image, where adminapi is not included?

would suggest moving the creation of the file to `ansible/tasks/setup-pgbouncer.yml` where the other two files are currently created

Reply:
okay moved this file creation from `admin-api.yml` to `setup-pgbouncer.yml`